src/utils/hasCacheTimedOut.ts

```diff
@@ -0,0 +1,9 @@
+import { CacheTimeouts } from '../types';
+
+const hasCacheTimedOut = (cacheTimeouts: CacheTimeouts, cacheIndex: string) => {
+  const { cacheSet, timeTillCacheInvalidate } = cacheTimeouts[cacheIndex];
+  if (Date.now() - cacheSet > timeTillCacheInvalidate) return true;
```

Comment:
Could turn this into a ternary

Reply:
i could actually just return the statement ```Date.now() - cacheSet > timeTillCacheInvalidate``` i think

Reply:
Yep good point - do that :)